src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task01.java

```diff
@@ -36,8 +35,10 @@ public static void main(String[] args) {
      * </ul>
      */
     static boolean isTopPlayer(int sizeTop, int positionPlayerByDamage) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return false;
+
+        if (positionPlayerByDamage>=sizeTop) {
+            return true;}
+        else
+            return false;
```

Comment:
"Violation: There isn't separating white spaces (OR line break) for curly brake
Solution: Add white spaces"